metadata.yaml

```diff
@@ -6,7 +6,7 @@ description: |
   A client provides a way to sync or act on changes in a remote stream.
 summary: |
   Simple Streams repository
-series: [bionic]
+series: [bionic,focal]
```

Comment:
Would you mind terribly using a list indent or add a space?

Reply:
agreed! thanks for catching this.